concepticondata/conceptlists/Nurse-1975-1038.tsv

```diff
@@ -702,7 +702,7 @@ Nurse-1975-1038-689	689	threaten	v.	threaten	(ku)tisha	1192	THREATEN
 Nurse-1975-1038-690	690	hen, fowl	n.	hen, fowl	kuku	265	FOWL
 Nurse-1975-1038-691	691	shadow, shade	n.	shadow, shade	kivuli	1388	SHADE
 Nurse-1975-1038-692	692	virgin (bride), girl	n.	virgin (bride), girl//virgin(bride), girl	mwali		
-Nurse-1975-1038-693	693	youth	n.	youth	kijana		
+Nurse-1975-1038-693	693	youth	n.	youth	kijana	1304	CHILD (YOUNG HUMAN)
```

Comment:
We'd need to double-checkl if the gloss *kijana* refers to the abstract concept or to the concept refering to a young person.